client/components/spinner/index.jsx

```diff
@@ -30,30 +30,33 @@ const isSVGCSSAnimationSupported = ( () => {
 export default class Spinner extends PureComponent {
 	static propTypes = {
 		className: React.PropTypes.string,
+		delay: React.PropTypes.number,
 		size: React.PropTypes.number,
-		duration: React.PropTypes.number
+		counterClockwise: React.PropTypes.bool,
 	};
 
 	static instances = 0;
 
 	static defaultProps = {
+		delay: 100,
```

Comment:
This will make each `Spinner` instance which does't specify the `delay` prop delayed by 100 milliseconds. Do we want that?